app/config/collections.php

```diff
@@ -4471,6 +4471,39 @@
                 'array' => false,
                 'filters' => [],
             ],
+            [
+                '$id' => ID::custom('status'),
+                'type' => Database::VAR_BOOLEAN,
+                'format' => '',
+                'size' => 0,
+                'signed' => true,
+                'required' => false,
+                'default' => true,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('errors'),
+                'type' => Database::VAR_INTEGER,
+                'format' => '',
+                'size' => 0,
+                'signed' => true,
+                'required' => false,
+                'default' => 0,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('logs'),
```

Comment:
Seing combination of `errors` and `logs` makes me think of `stderr` and `stdout`.

I would rename to `logs` and `attempts` - we call them like this in `certificates` collection